routes/POST-profile.js

```diff
@@ -0,0 +1,46 @@
+const s3Client = require('../config/s3');
+const { v4: uuid } = require('uuid');
+
+module.exports = (app, db) => {
+  app.post('/profile', (req, res) => {
+    var file;
+    var fileExtension;
+    var fileKey;
+    const sesh = req.session;
+    if (!sesh.email) {
+        return res.redirect('/?tm=You Aren\'t Logged In!&ts=false');
+    }
+    if (!req.files || Object.keys(req.files).length === 0) {
+      return res.redirect('/profile?tm=No files were uploaded&ts=false');
+    }
+    file = req.files.image;
+    fileExtension = file.name.substring(file.name.lastIndexOf(".") + 1);
+    fileKey = uuid() + "." + fileExtension;
+    var params = {
+      Bucket: process.env.BUCKET_NAME,
+      Key: fileKey,
+      Body: file.data
+    };
+    s3Client.upload(params, function(err, data) {
+      if (err) {
+          throw err;
```

Comment:
If an error occurs, then it should be gracefully handled and the user should be redirected to an appropriate location with the relevant error messages.